src/routes/1st/index.svelte

```diff
@@ -0,0 +1,322 @@
+<script context="module">
+  const confetti = import.meta.globEager('$lib/1st/confetti/*.ts');
+</script>
+
+<script lang="ts">
+  import ToggleConfetti from '$lib/1st/confetti/toggle-confetti.svelte';
+  import CommonHero from '$lib/components/sections/common-hero.svelte';
+  import Spacing from '$lib/components/sections/spacing.svelte';
+  import { Confetti } from 'svelte-confetti';
+
+  const colorArray: string[] = Object.keys(confetti).map((key) => confetti[key].default);
+</script>
+
+<ToggleConfetti>
+  <div
+    style="position: fixed; top: -50px; left: 0; height: 100vh; width: 100vw; display: flex; justify-content: center; overflow: hidden;"
+  >
+    <Confetti
+      size={20}
+      x={[-5, 5]}
+      y={[0, 0.1]}
+      delay={[500, 2000]}
+      {colorArray}
+      infinite
+      duration={5000}
+      amount={200}
+      fallDistance="100vh"
+    />
+  </div>
+</ToggleConfetti>
+
+<Spacing --min="175px" --med="200px" --max="200px" />
+
+<CommonHero>
+  <h2 slot="headline" class="size-lg">First Contributions</h2>
+  <p slot="text" class="size-sm">
+    <code>acmcsuf.com/1st</code> aims to simplify and guide the way beginners make their first
+    contribution. If you are looking to make your first contribution, follow the steps below.
+    <br />
+    If you're not comfortable with command line,
+    <a
+      href="https://github.com/firstcontributions/first-contributions/blob/main/README.md#tutorials-using-other-tools"
+      >here are tutorials using GUI tools</a
+    >.
+    <br />
+    If you don't have Git on your machine,
+    <a href="https://help.github.com/articles/set-up-git/">install it</a>.
+    <br />
+    Last updated September 9th, 2022
+  </p>
+</CommonHero>
+
+<Spacing --med="64px" />
+
+<main>
+  <section id="welcome">
+    <h2 class="headers">Welcome to ACM!</h2>
+
+    <p>
+      acmCSUF provides a social online platform, Discord, that brings passionate Titans together to
+      engage in multiple conversations regarding our community, the tech field, and many more. Join
+      <a href="/discord">our Discord server</a> to learn more about future events, meet new people, and
+      get involved with the community!
+    </p>
+  </section>
```

Comment:
i dont really like the mix of center align and left align + the width of these containers is not consistent

Reply:
![image](https://user-images.githubusercontent.com/8981287/189440090-38392357-2c1d-430d-9f45-000ebde789da.png)